src/plugins/data/public/ui/dataset_navigator/_dataset_navigator.scss

```diff
@@ -3,8 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 .datasetNavigator {
-  min-width: 350px;
+  min-width: 365px;
+  max-height: 60vh;
+  overflow-y: auto;
   border-bottom: $euiBorderThin !important;
+  scrollbar-width: thin;
+  scrollbar-color: $euiColorMediumShade $euiColorLightestShade;
```

Comment:
Can you add scroll to the panel and not the whole context window. Also oui has a scroll mixin that adds the correctly styled scrollbar: https://arc.net/l/quote/merikdgx

Reply:
gotcha, did you want me to remove this because i saw you have an update or just fast follows?